src/components/LHNOptionsList/LHNOptionsList.js

```diff
@@ -164,20 +142,17 @@ function LHNOptionsList({
 
     return (
         <View style={style || styles.flex1}>
-            <FlatList
+            <FlashList
                 indicatorStyle="white"
                 keyboardShouldPersistTaps="always"
                 contentContainerStyle={contentContainerStyles}
-                showsVerticalScrollIndicator={false}
                 data={data}
                 testID="lhn-options-list"
                 keyExtractor={keyExtractor}
-                stickySectionHeadersEnabled={false}
                 renderItem={renderItem}
-                getItemLayout={getItemLayout}
-                initialNumToRender={20}
-                maxToRenderPerBatch={5}
-                windowSize={5}
+                estimatedItemSize={optionMode === CONST.OPTION_MODE.COMPACT ? variables.optionRowHeightCompact : variables.optionRowHeight}
+                extraData={[currentReportID]}
```

Comment:
This is missing other data, e.g. `preferredLocale` which caused the list not to re-render on language change